apps/wallet/src/components/settings/DisasterRecoveryCard.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <VCard :loading="loading">
+    <VCardTitle>
+      {{ $t(`app.disaster_recovery_card_title`) }}
+    </VCardTitle>
+
+    <VCardText v-if="!loading && error" class="pb">
+      <VAlert type="error" variant="tonal" density="compact">
+        {{ $t('app.data_load_error') }}
+      </VAlert>
+    </VCardText>
+    <VCardText v-else-if="!!systemInfo" class="pb">
+      <VList v-if="systemInfo.disaster_recovery[0]" lines="two" class="bg-transparent">
+        <VListItem class="px-0">
+          <VListItemTitle class="font-weight-bold">{{ $t(`terms.user_group`) }}: </VListItemTitle>
+          <VListItemSubtitle>
+            {{ systemInfo.disaster_recovery[0].user_group_name[0] || $t('terms.unknown') }}
+          </VListItemSubtitle>
+        </VListItem>
+        <VListItem class="px-0">
+          <VListItemTitle class="font-weight-bold"
+            >{{ $t(`terms.user_group_id`) }}:
+          </VListItemTitle>
+          <VListItemSubtitle>
+            {{ systemInfo.disaster_recovery[0].committee.user_group_id }}
```

Comment:
nit: would be nice to have here the copy to clipboard btn

Reply:
added 👍